corehq/apps/integration/payments/views.py

```diff
@@ -86,6 +88,43 @@ def get_queryset(self):
         query = self._apply_filters(query)
         return query
 
+    def get_context_data(self, **kwargs):
+        context_data = super().get_context_data(**kwargs)
+
+        context_data['user_or_cases_verification_statuses'] = self._get_user_or_cases_verification_status(
+            context_data['page_obj'].object_list
+        )
+
+        return context_data
+
+    def _get_user_or_cases_verification_status(self, object_list):
+        if not self.kyc_config:
+            return {}
+
+        user_or_case_ids = self._get_user_or_case_ids(object_list)
+        kyc_users = self.kyc_config.get_kyc_users_by_ids(user_or_case_ids)
+        return {
+            kyc_user.user_id: kyc_user.kyc_verification_status
+            for kyc_user in kyc_users
+        }
+
+    @cached_property
+    def kyc_config(self):
+        try:
+            return KycConfig.objects.get(domain=self.request.domain)
+        except KycConfig.DoesNotExist:
+            return None
+
+    @staticmethod
+    def _get_user_or_case_ids(object_list):
+        user_or_case_ids = []
+        for commcare_payment_case_details in object_list:
```

Comment:
Is `object_list` a list of dictionaries? Is `commcare_payment_case_details` a dictionary? (Again, I miss type hints.) It would be nice to know what these variables look like. ... I'm just complaining. I'm not asking for this to be changed, at least not now, in this PR.